src/NewTools-Inspector/StInspectorModel.class.st

```diff
@@ -85,7 +85,7 @@ StInspectorModel >> inspectedObject: anObject [
 { #category : #accessing }
 StInspectorModel >> labelString [
 	
-	^ self descriptionString truncateWithElipsisTo: 30
+	^ self descriptionString
```

Comment:
The problem when we don't truncate the string is that the title is cutted without any dots that indicates that the name continues.

<img width="698" alt="Capture d’écran 2022-11-03 à 09 03 11" src="https://user-images.githubusercontent.com/33934979/199671900-931b9589-dd65-4996-bf4e-491ea51b16ee.png">

Reply:
I also do not think it is a good change. Maybe it would be nice to have a configurable truncation of Strings, depending on the space available to display it. But anyway, it would be another PR.
I will close this one. Feel free to open a new one.